packages/server/tests/localAcceptance.env

```diff
@@ -27,7 +27,7 @@ LIMIT_DURATION=90000
 SERVER_REQUEST_TIMEOUT_MS=60000
 MEMWATCH_ENABLED=true
 WRITE_SNAPSHOT_ON_MEMORY_LEAK=false
-HBAR_RATE_LIMIT_TINYBAR=5000000000# 50 HBARs
-HBAR_RATE_LIMIT_DURATION=80000# 80 seconds
-HBAR_RATE_LIMIT_BASIC=4000000000# 40 HBARs
+HBAR_RATE_LIMIT_TINYBAR=9000000000# 90 HBARs
+HBAR_RATE_LIMIT_DURATION=86400000# 24 hours
+HBAR_RATE_LIMIT_BASIC=6000000000# 60 HBARs
```

Comment:
**Note for Reviewers:** 
- The values of HBAR_RATE_LIMIT_TINYBAR has been increased due to the addition of more exhaustion tests in the hbar limiter. 

- Increasing HBAR_RATE_LIMIT_DURATION prevents the limiter from resetting during tests.

- Increasing HBAR_RATE_LIMIT_BASIC prevents api_batch1 from becoming flaky and encountering HBAR rate limit errors.

Reply:
@quiet-node  I thought we can add specific values for the hbar limiter acceptance tests directly in the command which runs this suite. This way we will mimic the actual behaviour of the limiter, which is also the goal of acceptance tests. So instead of just bumping the values all the time, we can just restart the limiter maybe just set a higher duration but not such high values. wdyt?

Reply:
@konstantinabl Thank you for your comment! I understand your concern. Currently, the duration is set to 24 hours by default, and the total limit is 8,000 HBARs. I hope this adjustment will be the final time we modify these variables. However, if you prefer, I can certainly revert it back to the command. Please let me know!